src/AccessibilityInsights.CommonUxComponents/Controls/ProgressRingControl.xaml.cs

```diff
@@ -71,28 +71,6 @@ private static void WithSoundPropertyChanged(DependencyObject d, DependencyPrope
         private bool isPlayingSound;
         private bool withSound;
 
-        /// <summary>
```

Comment:
Removing this allows a couple of using blocks to be removed from the top of this class